doc/performance_counters.rst

```diff
@@ -3,144 +3,48 @@
-+----------+----------------+----------------------------------------------------------------+
-| PCCR12   | **ST_EXT**     | Number of memory stores to EXT executed. Misaligned accesses   |
-|          |                | are counted twice. Every non-TCDM access is considered         |
-|          |                | external (PULP only)                                           |
-+----------+----------------+----------------------------------------------------------------+
-| PCCR13   | **LD_EXT_CYC** | Cycles used for memory loads to EXT. Every non-TCDM access is  |
-|          |                | considered external (PULP only)                                |
-+----------+----------------+----------------------------------------------------------------+
-| PCCR14   | **ST_EXT_CYC** | Cycles used for memory stores to EXT. Every non-TCDM access is |
-|          |                | considered external (PULP only)                                |
-+----------+----------------+----------------------------------------------------------------+
-| PCCR15   | **TCDM_CONT**  | Cycles wasted due to TCDM/log-interconnect contention          |
-|          |                | (PULP only)                                                    |
-+----------+----------------+----------------------------------------------------------------+
-| PCCR31   | **ALL**        | Special Register, a write to this register will set all        |
-|          |                | counters to the supplied value                                 |
-+----------+----------------+----------------------------------------------------------------+
-
-In the FPGA, RTL simulation and Virtual-Platform there are individual counters for each event type, i.e. PCCR0-30 each represent a separate register. To save area in the ASIC, there is only one counter and one counter register. Accessing PCCR0-30 will access the same counter register in the ASIC. Reading/writing from/to PCCR31 in the ASIC will access the same register as PCCR0-30.
-
-:numref:`pcer` shows how events are first masked with the PCER register and then ORed together to increase the one performance counter PCCR.
-
-.. figure:: images/pcer.png
-   :name: pcer
-
-   Events and PCCR, PCMR and PCER on the ASIC.
+Ibex implements performance counters according to the RISC-V Privileged Specification, draft version 1.11 (see Hardware Performance Monitor, Section 3.1.11).
+The performance counters are placed inside the Control and Status Registers (CSRs) and can be accessed with the ``CSRRW(I)`` and ``CSRRS/C(I)`` instructions.
+
+Ibex implements the clock cycle counter ``mcycle(h)``, the retired instruction counter ``minstret(h)``, as well as the 29 event counters ``mhpmcounter3(h)`` - ``mhpmcounter31(h)``. 
+
+All counters have a width of 64 bits. The lower 32 bits can be accessed through the base register, whereas the upper 32 bits are accessed through the ``h``-register.
```

Comment:
Maybe make the detailed discussion of the counters a separate subsection?

Reply:
Done